pom.xml

```diff
@@ -45,6 +45,9 @@
     <!-- Properties -->
     <properties>
 
+        <!-- Galleon feature pack -->
+        <product.release.name>WildFly Camel</product.release.name>
```

Comment:
Do we need this? Can we not use `${project.name}`? Or at a minimum set `product.release.name` to `project.name`.

Reply:
IIRC, this string appears on the console, when the server starts. It needs to be different and easily maintainable for Fuse@EAP. Would `${project.name}` work well also for the product?

Reply:
Ok, fair enough. On the product side we wont want to use `project.name` so I guess we need this separate property.